fb-messenger-dark.css

```diff
@@ -51,6 +51,12 @@ input::-webkit-input-placeholder {
 ._33p7 {
 	background-color: rgba(30, 30, 30, 0.9)	!important;
 }
+/* highlighted search results */
+.__in {
+	background-color: rgba(0, 0, 0, 0.3);
+	box-shadow: 2px 0 rgba(0, 0, 0, 0.3), -2px 0 rgba(0, 0, 0, 0.3);
```

Comment:
I'm pretty sure you'll want these to be `!important` for this to actually work consistently when installed as a userscript.

Reply:
Ah yeah, you're probably right.

The thing is, I currently have these styles without `!important` in the following userscript and it still overrides the default styles. I recall [this having been a problem before](https://github.com/auscompgeek/fb-messenger-dark/pull/17#issuecomment-288868948). Do you know what causes it?

```js
// ==UserScript==
// @name        Custom Messenger Changes
// @namespace   https://github.com/HatScripts
// @version     DEV
// @author      HatScripts
// @match       http*://www.messenger.com/*
// @run-at      document-start
// @require     https://rawgit.com/HatScripts/UserscriptHelpers/master/applyCss.min.js
// ==/UserScript==

applyCss(`
._2swd._kmc {
	padding-bottom: 9px !important;
}

.__in {
	background-color: rgba(0, 0, 0, 0.3);
	box-shadow: 2px 0 rgba(0, 0, 0, 0.3), -2px 0 rgba(0, 0, 0, 0.3);
	padding: 1px 0 2px 0;
}
`);
```

Reply:
There's a potential race condition where messenger.com's JS finishes before userscripts might have a chance to run.